package.json

```diff
@@ -71,7 +71,7 @@
     "eslint-plugin-import": "^2.7.0",
     "eslint-plugin-prettier": "^2.3.1",
     "husky": "^0.14.3",
-    "jest": "^21.2.1",
+    "jest": "^23.5.0",
```

Comment:
This change seems unrelated. Is this needed for anything?

Reply:
When I ran `yarn test` (after `yarn install`) it was failing with:

```bash
$ yarn test
yarn run v1.5.1
$ jest --runInBand
(node:96019) UnhandledPromiseRejectionWarning: Error: Cannot find module './node_modules/jest-cli'
    at Function.Module._resolveFilename (module.js:547:15)
    at Function.Module._load (module.js:474:25)
    at Module.require (module.js:596:17)
    at require (internal/module.js:11:18)
    at getJest (./node_modules/jest/node_modules/jest-cli/build/cli/get_jest.js:23:12)
    at _getRunCLIFn (./node_modules/jest/node_modules/jest-cli/build/cli/index.js:140:69)
    at Object.<anonymous> (./node_modules/jest/node_modules/jest-cli/build/cli/index.js:41:22)
    at Generator.next (<anonymous>)
    at step (./node_modules/jest/node_modules/jest-cli/build/cli/index.js:45:2466)
    at ./node_modules/jest/node_modules/jest-cli/build/cli/index.js:45:2696
(node:96019) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:96019) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
✨  Done in 0.27s.
```

I updated the version of `jest` to latest and everything worked as expected.

Reply:
For reference, I'm running `node v8.11.4` (lts/carbon).